datastore/datastore_test.go

```diff
@@ -3081,6 +3081,103 @@ func TestPutMultiTypes(t *testing.T) {
 	}
 }
 
+func TestGetWithReadTime(t *testing.T) {
+	type Ent struct {
+		A int
+		B string
+	}
+
+	k := NameKey("testKind", "testReadTime", nil)
+	e := &pb.Entity{
+		Key: keyToProto(k),
+		Properties: map[string]*pb.Value{
+			"A": {ValueType: &pb.Value_IntegerValue{IntegerValue: 1}},
+			"B": {ValueType: &pb.Value_StringValue{StringValue: "one"}},
+		},
+	}
+	fakeClient := &fakeDatastoreClient{
+		lookup: func(*pb.LookupRequest) (*pb.LookupResponse, error) {
+			return &pb.LookupResponse{
```

Comment:
Could you capture the lookup request's read time here to validate that the client is passing it through correctly?

Reply:
Ditto below.